index.js

```diff
@@ -97,6 +97,7 @@ Toolkit.run(async tools => {
     newVersion = execSync(`npm version --git-tag-version=false ${version}`).toString().trim()
     newVersion = `${process.env['INPUT_TAG-PREFIX']}${newVersion}`
     console.log('new version:', newVersion)
+    console.log(`::set-output name=newTag::${newVersion}`)
```

Comment:
@falko189 can you point to the official documentation of this?

Reply:
https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/metadata-syntax-for-github-actions#outputs

https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-output-parameter

I think a console log is enough? or should it be inside the docker file as actual echo

Reply:
I also realised that in case we merge this I should update the doc to reflect the change.

Reply:
I am not sure, but I hope you tested it?!

Reply:
Not sure how to test this on public repos, I have worked with actions only on private repos 🤔

Reply:
doesnt matter if public or private repo. the action is public and can be used. so you should use your branch as action and test your behavior if you get the output you want.

Reply:
Closing this to perform testing first :D